applications/CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ add_holohub_application(object_detection_torch)
 
 add_holohub_application(openigtlink_3dslicer DEPENDS OPERATORS openigtlink)
 
+add_holohub_application(privacy)
```

Comment:
Let's rename this application to something more meaningful. Maybe video_anonymizer?

Reply:
along the same lines, how about `video_deidentification`

Reply:
Renamed app